local-volume/README.md

```diff
@@ -109,27 +109,53 @@ $ export KUBE_FEATURE_GATES="PersistentLocalVolumes=true,VolumeScheduling=true,M
 
 #### Option 1: GCE
 
+GCE clusters brought up with kube-up.sh will automatically format and mount the
+requested Local SSDs, so you can deploy the provisioner with the pre-generated
+deployment spec and skip to [step 4](#step-4-create-local-persistent-volume-claim),
+unless you want to customize the provisioner spec or storage classes.
+
 ##### Pre-1.9
 
 ``` console
 $ NODE_LOCAL_SSDS=<n> cluster/kube-up.sh
+$ kubectl create -f provisioner/deployment/kubernetes/admin_account.yaml
+$ kubectl create -f provisioner/deployment/kubernetes/gce/provisioner_generated_gce_pre19.yaml
 ```
 
 ##### 1.9+
 
 ``` console
 $ NODE_LOCAL_SSDS_EXT=<n>,<scsi|nvme>,fs cluster/kube-up.sh
-# This handles creating the StorageClasses
-$ kubectl create -f examples/deployment/kubernetes/gce/class-local-ssds.yaml
+$ kubectl create -f provisioner/deployment/kubernetes/admin_account.yaml
+$ kubectl create -f provisioner/deployment/kubernetes/gce/class-local-ssds.yaml
+$ kubectl create -f provisioner/deployment/kubernetes/gce/provisioner_generated_gce_19.yaml
 ```
 
 #### Option 2: GKE
 
+GKE clusters will automatically format and mount the
+requested Local SSDs, so you can deploy the provisioner with the pre-generated
+deployment spec and skip to [step 4](#step-4-create-local-persistent-volume-claim),
+unless you want to customize the provisioner spec or storage classes.
+
+##### Using local-ssd-count option
 ``` console
 $ gcloud container cluster create ... --local-ssd-count=<n> --enable-kubernetes-alpha
 $ gcloud container node-pools create ... --local-ssd-count=<n>
+$ kubectl create -f provisioner/deployment/kubernetes/admin_account.yaml
+
 # If running K8s 1.9+, also create the StorageClasses
-$ kubectl create -f examples/deployment/kubernetes/gce/class-local-ssds.yaml
+$ kubectl create -f provisioner/deployment/kubernetes/gce/class-local-ssds.yaml
+$ kubectl create -f provisioner/deployment/kubernetes/gce/provisioner_generated_gce_pre19.yaml
```

Comment:
should this be not "pre"19

Reply:
If they're using the old --local-ssd-count, then they need to use the pre1.9 config which looks at /mnt/disks.

Yeah, the naming is confusing.  What do you think of naming the file "v1" or "v2" instead of "pre19" and "19"

Reply:
v1 or v2 doesn't seem great to me, they don't really mean anything on their own and it implies you can always use v2 because its newer/better.

Then again I can't think of anything better. I'm not against pre19 and 19. Maybe add a comment for that particular call?

Reply:
Pre 1.9 is confusing because in GKE you could still use the old count option with 1.9+.